### PR TITLE
feat: testkube logs service chart

### PR DESCRIPTION
**Pull request by exu**
Checklist:

* [ ] Any new values are backwards compatible and/or have sensible default.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/kubeshop/testkube-cloud-charts/blob/master/community/CONTRIBUTING.md).
* [ ] My CI is green.

### CODE REVIEW

File: charts/testkube-enterprise/values.yaml [hunk 1]
```diff
@@ -238,6 +238,27 @@ testkube-worker-service:
   image:
     repository: testkubeenterprise/testkube-enterprise-worker-service
     tag: 1.9.0-rc-e5167f6a
+
+testkube-logs-service:
```
Comment (ypoplavs) [inline charts/testkube-enterprise/values.yaml:242]: If you want to include logs service into Enterprise chart, you also need to add it to `Chart.yaml`

File: charts/testkube-logs-service/values.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  labels: {}
+  # -- Additional annotations to add to the gRPC Ingress resource
+  annotations: {}
+  # kubernetes.io/ingress.class: nginx
+  # -- Max payload size for proxied gRPC API
+  maxPayloadSize: 16m
+  # -- Hostname for which to create rules and TLS certificates (if omitted, the host will be generated using the global subdomain and `domain` values)
+  host: ""
+
+websocketsIngress:
```
Comment (ypoplavs) [inline charts/testkube-logs-service/values.yaml:71]: I think there's no need for `websocketsIngress` and `restIngress` here, as you only create `grpcIngress`

File: charts/testkube-logs-service/values.yaml [hunk 3]
```diff
@@ -0,0 +1,197 @@
+# Default values for testkube-cloud-api.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  # -- Domain under which to create Ingress rules
+  domain: ""
+  # -- UI subdomain which get prepended to the domain
+  uiSubdomain: "cloud"
```
Comment (ypoplavs) [inline charts/testkube-logs-service/values.yaml:9]: these global values are not needed here: `uiSubdomain`, `restApiSubdomain`, `grpcApiSubdomain`, `websocketApiSubdomain` as they are not used in the chart.

File: charts/testkube-logs-service/templates/_helpers.tpl [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+{{- define "testkube-log-service.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
+{{- default (include "testkube-log-service.fullname" .) .Values.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
+{{/*
+Get REST Ingress host
```
Comment (ypoplavs) [inline charts/testkube-logs-service/templates/_helpers.tpl:80]: Not needed if you don't create REST Ingress host, the same goes for  Websockets Ingress host,

File: charts/testkube-logs-service/values.yaml (hunk 3 above)
Comment (exu) [reply charts/testkube-logs-service/values.yaml:9]: grpcApiSubdomain is used removed rest

File: charts/testkube-enterprise/values.yaml (hunk 1 above)
Comment (ypoplavs) [inline charts/testkube-enterprise/values.yaml:242]: also please duplicate these values to a `local-values.yaml` file that is used for simplified Enterprise installation.

File: charts/testkube-logs-service/templates/_helpers.tpl [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+*/}}
+{{- define "testkube-log-service.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "testkube-log-service.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+MinIO selector labels
+*/}}
+{{- define "testkube-log-service.minio.selectorLabels" -}}
```
Comment (ypoplavs) [inline charts/testkube-logs-service/templates/_helpers.tpl:54]: these are not used anywhere, you may remove them

File: charts/testkube-enterprise/values.yaml (hunk 1 above)
Comment (exu) [reply charts/testkube-enterprise/values.yaml:242]: done :)

File: charts/testkube-logs-service/templates/ingress-grpc.yaml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    {{- with .Values.grpcIngress.annotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+spec:
+  {{- if and .Values.ingress.className (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
+  ingressClassName: {{ .Values.ingress.className }}
+  {{- end }}
+  tls:
+    - hosts:
+        - {{ include "testkube-log-service.ingress.grpcHost" . | quote }}
```
Comment (ypoplavs) [inline charts/testkube-logs-service/templates/ingress-grpc.yaml:43]: The Ingress will point to `agent....`. is that right?

Comment (exu) [reply charts/testkube-logs-service/templates/ingress-grpc.yaml:43]: yes I'm adding this one on path route of the agent - to no introduce another level of complexity. 

Ideally we should have API Gateway here to just add it - but here ingress will work for me like gateway.

**Review by ypoplavs — APPROVED** [review]

**Pull request merged**